docs/tendermint-rpc.md

```diff
@@ -0,0 +1,210 @@
+# Tendermint RPC
+
+[Tendermint RPC](https://docs.tendermint.com/v0.34/rpc/#) can be used to query application's state and/or subscribe to Tendermint WebSocket events.   
+The default RPC listen address is tcp://0.0.0.0:26657. See the [docs](https://docs.tendermint.com/v0.34/rpc/#) to learn how to configure and customize.
+
+## WebSocket Events
+
+Full list of Tendermint WebSocket events can be found [here](https://pkg.go.dev/github.com/tendermint/tendermint/types#pkg-constants).
+
+### Subscribe
+
+Subscribing to Tendermint WebSocket events can be done using any WS clients like [wscat](https://github.com/websockets/wscat), [Postman](https://www.postman.com/), etc. The following is an example of subscribing to a new `tx` (transaction) event using wscat:
+
+1. Connect to the server:
+
+```bash
+wscat -c ws://localhost:26657/websocket
```

Comment:
Please replace `localhost` with `<node-address>` (generic param).
Also please provide an example for one of open Test Net and Main Net nodes (Observers), for example from https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/deployment/persistent_chains/testnet-2.0/testnet-2.0-csa-endpoints.md and https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/deployment/persistent_chains/main-net/main-net-csa-endpoints.md.